packages/fxa-auth-server/lib/routes/subscriptions/account.ts

```diff
@@ -43,5 +45,16 @@ export async function sendFinishSetupEmailForStubAccount({
       ...metricsContext,
       token,
     });
+
+    if (metricsContext && subscriptionAccountReminders) {
+      await subscriptionAccountReminders.create(
```

Comment:
I think MetricsContext should be optional in this case. The email template shouldn't append those values if they are undefined. For some background, metrics context is what we use to follow a user through a signup/signin flow.